src/components/NamespaceDropdown.tsx

```diff
@@ -21,6 +21,7 @@ export class NamespaceDropdown extends React.PureComponent<NamespaceListType, {}
   }
 
   handleSelectNamespace = (namespace: string) => this.props.onSelect({ name: namespace });
+  onToggle = (isOpen: boolean) => isOpen && this.props.refresh();
```

Comment:
Shouldn't this be `handleToggle` instead of `onToggle`

Reply:
Yes, makes sense.
Fixed.